### ISSUE TITLE
Error in Mqtt.js

### ISSUE THREAD

**Issue by pierbettassa** [lorenzo-pizzari/hapi-mqtt#2]
Hi,
I think that line 21 in Mqtt.js contains an error:
```
client.on('error', (err) => {
                server.log(['hapi-Mqtt', 'info'], error.toString());
                reject(err)
            });
```

should be (error -> err):
```

client.on('error', (err) => {
                server.log(['hapi-Mqtt', 'info'], err.toString());
                reject(err)
            });
```

**Comment by lorenzo-pizzari** [comment]
Thanks for the note, should be a typo in the refactoring process. We'll fix ASAP

**Comment by lorenzo-pizzari** [comment]
@pierbettassa i've released the v1.0.1 on npm fixing this issue
